integration-tests/emulated/src/mock/para_chains.rs

```diff
@@ -63,7 +57,7 @@ pub mod spiritnet {
 				balances: accounts::init_balances()
 					.iter()
 					.cloned()
-					.map(|k| (k, ED * 4096))
+					.map(|k| (k, ED * 1_000_000_000_000_000_000))
```

Comment:
Can we use in terms of `KILT`s?

Reply:
Yes thats a good point: [3005b0d](https://github.com/KILTprotocol/kilt-node/pull/655/commits/3005b0d19cb1be373a81dd0b84dd1cd5402a9e52)